src/components/Dropdown/V2/Dropdown.Renderer.tsx

```diff
@@ -282,7 +305,7 @@ class Renderer extends React.PureComponent<any> {
     // It may feel "wrong"... But, this is FAR cheaper than
     // relying on React to do it.
 
-    // That is because we're doing with a single (more or less) calculcation
+    // That is because we're doing with a single (more or less) calculation
```

Comment:
@tinkertrain LOL thank you for the typo fix <3

Reply:
hehe!